### DISCUSSION TITLE
Fix the order of lists

### DISCUSSION THREAD

**Discussion by ddancn** [community/community#56194] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I kind of want to sort the order of lists by myself, or at least remember the currently selected sort plz. Now the default sort is always "last updated", so the order changes all the time, which confuses me everytime I open the star page.

I saw some discussions mention the same feedback, but it remains still. Please consider this feature!!

**Comment by pepperstreet** [comment]
Absolutely agree. custom or usual ordering options would be a great help. Over time you will have a lot of lists. A general A-Z order and ASC/DSC, maybe different dates like created/modified? And or a FILTER field on top? 
BTW, this reminds me on the old vanished youtube lists. The old list had a great functionality and they replaced it with a feature-less, almost useless successor... What were they thinking?

**Reply by andykim33** [reply]
Andykim

**Reply by MargieF7** [reply]
Please

**Reply by Leilakayed** [reply]
💎

**Comment by chanachai004** [comment]
พด

**Comment by andykim33** [comment]
a5a1caa5c358da3d6ad8a7ef60b2773997103d66

**Comment by andykim33** [comment]
a5a1caa5c358da3d6ad8a7ef60b2773997103d66

**Comment by andykim33** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I kind of want to sort the order of lists by myself, or at least remember the currently selected sort plz. Now the default sort is always "last updated", so the order changes all the time, which confuses me everytime I open the star page.
> 
> I saw some discussions mention the same feedback, but it remains still. Please consider this feature!!

**Comment by simplvod** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I kind of want to sort the order of lists by myself, or at least remember the currently selected sort plz. Now the default sort is always "last updated", so the order changes all the time, which confuses me everytime I open the star page.
> 
> I saw some discussions mention the same feedback, but it remains still. Please consider this feature!!

**Comment by Sammyreal1** [comment]
Approved

**Comment by niklas-kammertons** [comment]
Was about to write a similar issue, so I'll just append it here
**Bug: 'Sort by' resets itself**
---
I am trying to create lists with prefix numbers to sort them to my liking 01_ , 02_, ....
And I also have some Lists that do not belong to my structured numbered lists (e.g. fun project ideas)

<img width="915" alt="image" src="https://github.com/community/community/assets/150491000/7b9ced56-eeda-4ce4-ae6e-09fcb7e5e6df">

Once I create new Lists, the sorting feature A to Z is reset and sorts newest to oldest again. It would be nice if it could remember my choice forever and be persistent in the sorting order until I specifically change it again myself.

Maybe for public lists it would make sense to give the owner of the Lists a way to set the sorting order default to make a "list of lists" that can be shared in a structured manner to everyone.

**Comment by rafa-br34** [comment]
Another interesting idea would be to make the sort order persistent and also let the user choose a sort order while starring a repository.

**Reply by krzysztof293** [reply]
Gdzie są pieniadze

**Comment by robbertakse** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I kind of want to sort the order of lists by myself, or at least remember the currently selected sort plz. Now the default sort is always "last updated", so the order changes all the time, which confuses me everytime I open the star page.
> 
> I saw some discussions mention the same feedback, but it remains still. Please consider this feature!!

**Reply by Aslan11021983** [reply]
- - [ ] # > ~~@rileybroughten []()_****_~~

**Comment by haleygelato** [comment]
Agreed. Users should also be able to reorder the lists and set the order so that when others view their profile, they can see the customized order.
